dali/dfu/dfuwu.hpp

```diff
@@ -248,6 +251,7 @@ interface IConstDFUfileSpec: extends IInterface
     virtual StringBuffer &getRawDirectory(StringBuffer &str) const = 0;
     virtual StringBuffer &getRawFileMask(StringBuffer &str) const = 0;
     virtual bool getRemoteGroupOverride() const = 0;
+    virtual bool getPreserveCompression() const = 0;
```

Comment:
Is this ever used?  Isn't preservation an option of the copy rather than the file?